packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/AudienceView.tsx

```diff
@@ -25,21 +36,65 @@ export interface AudienceViewProps extends HasContainerId {
 export function AudienceView(props: AudienceViewProps): React.ReactElement {
 	const { containerId } = props;
 
-	// TODO: Post message requesting Audience summary
-	// TODO: Listen for Audience data updates
+	// POST Request for Audience Data
+	const getAudienceMessage: IDebuggerMessage = {
+		type: "GET_AUDIENCE_EVENT",
+		source: extensionMessageSource,
+		data: {
+			containerId,
+		},
+	};
+
+	const messageRelay = useMessageRelay();
+
+	const [clientId, setClientId] = React.useState<string | undefined>("");
+	const [audienceState, setAudienceState] = React.useState<AudienceClientMetaData[]>([]);
+	const [audienceHistory, setAudienceHistory] = React.useState<readonly AudienceChangeLogEntry[]>(
+		[],
+	);
 
 	React.useEffect(() => {
-		// TODO
-	}, [containerId]);
+		/**
+		 * Handlers for inbound messages related to Audience
+		 */
+		const inboundMessageHandlers: InboundHandlers = {
+			["AUDIENCE_EVENT"]: (untypedMessage) => {
+				const message: AudienceEventMessage = untypedMessage as AudienceEventMessage;
+
+				setClientId(message.data.clientId);
+				setAudienceState(message.data.audienceState);
+				setAudienceHistory(message.data.audienceHistory);
+
+				return true;
+			},
+		};
+
+		/**
+		 * Event handler for messages coming from the Message Relay
+		 */
+		function messageHandler(message: Partial<IDebuggerMessage>): void {
+			handleIncomingMessage(message, inboundMessageHandlers, {
+				context: loggingContext,
+			});
+		}
+
+		// Request state info for the newly specified containerId
```

Comment:
This comment was probably replaced by the one for the next line, I think this one can go away